### ISSUE TITLE
PF candidates with zero pT and large pseudorapidity

### ISSUE THREAD

**Issue by mandrenguyen** [cms-sw/cmssw#40626]
As discussed in #40032 there seem to be rare cases of PF candidates with anomalous kinematics.
While we now protect from known crashes in downstream modules, it would be good to understand the source of the problem to prevent future issues.

**Comment by mandrenguyen** [comment]
assign pf, reconstruction
@cms-sw/pf-l2

**Comment by swagata87** [comment]
to understand the source of the problem one might still need the original raw file where it crashed, i.e. a recipe to reproduce the problem ([this](https://github.com/cms-sw/cmssw/issues/40032#issuecomment-1308819142) does not work anymore ) ; without at least one example event where the issue occurs it's rather hard to gain any insight on this

**Comment by mandrenguyen** [comment]
Indeed.  We have the run number.  If we also had lumi/event info, we could track down the event.  @laurenhay Any chance you still have access to that info from the tests you ran for  #40088
?
